tests/test_update_atlas.py

```diff
@@ -0,0 +1,19 @@
+from brainatlas_api import update
+from click.testing import CliRunner
+
+
+def test_update():
+    update.update_atlas(atlas_name="allen_mouse_25um")
```

Comment:
Maybe better to use the example as it is much faster to download

Reply:
adding it

Reply:
Also, here it would also be important that apart from not failing running the code produce the correct results - in this case, maybe create a dummy folder with an outdated version in the name and check if it gets updated or not

Reply:
it downloads the latest version with the standard `BrainGlobeAtlas` functionality, so if something like that goes wrong it wouldn't be update's fault...